pkg/reconciler/resources/meta.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// MakeLabels returns a label map constructed from the union of labels of
+// the parent object and additional labels.
+func MakeLabels(obj metav1.Object, add map[string]string) map[string]string {
```

Comment:
Should this be `MergeLabels` then? Or `MergeMap` per the next comment.

Reply:
I kept them separate since, the actions were different, but sure :)